configuration/filesystems.md

```diff
@@ -0,0 +1,53 @@
+|:-----------|:-----------------------|:--------------------------------:|
+| EXT4       | 3.2                    |                                  |
+| XFS        | 3.2                    |                                  |
+| BTRFS      | 5.0                    | ✅                               |
+| ZFS        | N/A                    | ✅                               |
+| NFSv4      | N/A                    |                                  |
+
+
+
+
+### Ext4
+
+The Ext4 file system is well supported and an excellent default choice as a backing file system.  Ext4 supports volume sizes up to 1EiB and up to 4 Billion files, Gravwell extensively tests on Ext4.
+
+### XFS
+
+The XFS file system is extremely fast, well tested, and praised by kernel developers.  XFS supports a wide array of configuration options and to optimize the file system for specific storage device topology.
+
+### BTRFS
+
+The BTRFS file system has been a core part of the Linux kernel for over a decade, but due to some rocky starts and conservative warnings about stability early on in its life cycle it gets a bad rap.  Gravwell extensively tests the BTRFS file system in transparent compression topology and has found it to be exceedingly fast, memory efficient, and well supported.  While BTRFS is supported all the way back to Linux Kernel 3.2, 5.X and newer Kernels contain a highly optimized and stable code base.  Gravwell recommends BTRFS with ZSTD compression for a hot store when transparent compression is enabled and users want the best performance.
+
+### ZFS
+
+The ZFS file system has long been praised as **THE** next generation file system, it has a stable well maintained code base with robust documentation.  However, ZFS is in a bit of a strange situation in the Linux kernel in that many distributions do not natively support it and the Kernel maintainers believe it has an incompatible license.  ZFS also employs its own caching strategy that is not well blended with the Linux page cache, this means you need to manually tune the ZFS ARC cache and be aware that the ARC cache competes for memory with the Gravwell processes.  When memory gets tight ZFS will not free memory in the same way that BTRFS may.  That being said, the additional configuration options available in ZFS make it a good choice for cold storage when compression ratios are of the utmost importance.
+
+Gravwell recommends ZFS when transparent compression is desired for a cold storage tier and compression efficiency is more important than raw speed.  Setting the block size to 1MB and the compression system to zstd-10 can yield impression compression ratios that still perform well.  ZFS however is significantly slower than BTRFS when using transparent compression and a fast storage devices.  ZFS also does not support the ability to disable COW and compression on a per file basis, so ZFS will attempt to compress and fragment highly orthogonal data structures like well indexes.
```

Comment:
```suggestion
Gravwell recommends ZFS when transparent compression is desired for a cold storage tier and compression efficiency is more important than raw speed.  Setting the block size to 1MB and the compression system to zstd-10 can yield impression compression ratios that still perform well.  ZFS however is significantly slower than BTRFS when using transparent compression and a fast storage device.  ZFS also does not support the ability to disable COW and compression on a per file basis, so ZFS will attempt to compress and fragment highly orthogonal data structures like well indexes.
``` 

Made a correction here to make device singular, but have two other questions:
- Is "impression compression ratios" what you meant to write here?
- Should we spell out COW as copy-on-write? Don't know if this is too much jargon or if it's just fine to leave it...?